packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -0,0 +1,41 @@
+import {Component, h, Host, Listen, State} from '@stencil/core';
+
+export interface UpdateLiveRegionEventArgs {
+  region: string;
+  message: string;
+}
+
+/**
+ * The `atomic-aria-live` component notifies screen readers of changes in the search interface.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-aria-live',
+  shadow: false,
+})
+export class AtomicAriaLive {
+  @State() public regions: Record<string, string> = {};
+
+  @Listen('atomic/accessibility/updateLiveRegion', {target: 'document'})
```

Comment:
What happens if two components dispatch an aria live message at the same time, or very close to one another ?

Might be overkill today, but we could have some kind of debouncing in place, so that we don't change the messages too fast before the screen reader has a chance to do it's job ?

Or do screen reader already handle this gracefully ? (aka: It will finish it's current sentence before announcing the next one)

Reply:
In `polite` mode, the screen reader will finish the current sentence before announcing the update. This is the recommended approach, unless there is a critical update to announce.

Ref: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions

Reply:
As Sami mentioned, it waits for the previous sentence to be done (even if you changed the same region very quickly).